File: Assignment_15/Q5.c
///////////////////////////////////////////////////////////////////////////////////////////////
//
//  File name :     assignment15-Q5.c
//  Descreption :   accept no of rows and column from user and display below pattern
//  Author :        Vaishanvi D Shingare
//  Date :          28/04/2025

// input : iRow = 4     iCol = 4

/*  output : 1  2  3  4
             1  *  *  4
             1  *  *  4
             1  2  3  4
*/

#include<stdio.h>

void Pattern(int iRow , int iCol)
{
    int i = 0;
    int j = 0;

    for(i = 1;i <= iRow; i++)
    {
        for(j = 1; j <= iCol; j++)
        {
            if((i == 1) || ( i == iRow) || (j == 1) || (j == iCol))
            {
                printf("%d\t",j);
            }
            else
            {
                printf("*\t");
            }
        }
        
        printf("\n");
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter numbers of rows and columns :");
    scanf("%d %d",&iValue1,&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}

File: Assignment_46/Q5.java
///////////////////////////////////////////////////////////////////////////////////////////////
//
//  File name :     assignment46-Q5.java
//  Descreption :   write java program which accepts nos of rows and colunm from user and display below patterns.
//  Author :        Vaishanvi D Shingare
//  Date :          28/01/2025

// Input 3  4   
/*   1       2       3       4
     5       6       7       8
     9       10      11      12
*/

import java.util.*;

class Pattern
{
    public void Display(int iRow, int iCol)
    {
        int i = 0, j = 0,iCnt = 1;

        for(i = 1 ; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++)
            {
                System.out.print(iCnt+"\t");
                iCnt++;
            }
            System.out.print("\n");
        }
    }
}

class Q5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter number of rows : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter number of columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1, iValue2);
    }
}